assume/strategies/flexable.py

```diff
@@ -7,6 +7,10 @@
 
 
 class flexableEOM(BaseStrategy):
+    """
+    A strategy that bids on the EOM-market.
```

Comment:
by placing two bids onto the market.
One for inflexable amount and a second one for flexable bidding amount at a higher price